src/libs/histogram.c

```diff
@@ -1476,6 +1495,100 @@ static void _lib_histogram_preview_updated_callback(gpointer instance, dt_lib_mo
   dt_control_queue_redraw_widget(d->scope_draw);
 }
 
+void _menuitem_preferences(GtkMenuItem *menuitem, dt_lib_module_t *self)
+{
+  GtkWidget *win = dt_ui_main_window(darktable.gui->ui);
+  GtkWidget *dialog = gtk_dialog_new_with_buttons(_("scopes settings"), GTK_WINDOW(win),
+                                       GTK_DIALOG_DESTROY_WITH_PARENT,
+                                       _("cancel"), GTK_RESPONSE_NONE, _("save"), GTK_RESPONSE_YES, NULL);
+  GtkWidget *area = gtk_dialog_get_content_area(GTK_DIALOG(dialog));
+  gtk_widget_set_size_request(area, 300, -1); // width is too narrow otherwise
+
+  GtkWidget *combo = dt_bauhaus_combobox_new_action(DT_ACTION(self));
+  dt_bauhaus_widget_set_label(combo, NULL, N_("display scopes"));
+  dt_bauhaus_combobox_add(combo, _("in the left sidebar"));
+  dt_bauhaus_combobox_add(combo, _("in the right sidebar"));
+
+  dt_lib_histogram_position_t position = DT_LIB_HISTOGRAM_RIGHT;
+  if(dt_conf_key_exists("plugins/darkroom/histogram/position"))
+    position = dt_conf_get_int("plugins/darkroom/histogram/position");
+  dt_bauhaus_combobox_set(combo, position);
+
+  gtk_box_pack_start(GTK_BOX(area), combo, TRUE, TRUE, 0);
+
+#ifdef GDK_WINDOWING_QUARTZ
+  dt_osx_disallow_fullscreen(dialog);
+#endif
```

Comment:
in order to have this you must add

```c
#ifdef GDK_WINDOWING_QUARTZ
#include "osx/osx.h"
#endif
```

to includes.

Reply:
done, thanks !